build/charts/antrea/templates/antctl/clusterrole.yaml

```diff
@@ -26,9 +26,12 @@ rules:
       - networkpolicystats
       - antreaclusternetworkpolicystats
       - antreanetworkpolicystats
+      - nodeiplatencystats
     verbs:
       - get
       - list
+      - create
+      - update
```

Comment:
This is not needed for this role (`antctl`), please be careful with RBAC

Reply:
OK, I will remove them.